alcs-api/src/application/application-status/application-status.service.ts

```diff
@@ -31,6 +31,15 @@ export class ApplicationStatusService {
     return await this.applicationStatusRepository.save(applicationEntity);
   }
 
+  async fetchStatusId(statusCode: string) {
```

Comment:
lets return the whole object here, so if ever need other properties we have everything in place

Reply:
Done!